lib/rules/index.js

```diff
@@ -202,6 +203,7 @@ export default {
   'no-route-action': norouteaction,
   'no-scope-outside-table-headings': noscopeoutsidetableheadings,
   'no-shadowed-elements': noshadowedelements,
+  'no-splattributes-with-class': nosplattributeswithclass,
```

Comment:
we need to run `npm run update` to adjust readme.

<img width="937" alt="image" src="https://github.com/user-attachments/assets/a872d496-ceb7-4f42-9958-35b3fda482ff" />

Reply:
@zfir nope